apps/hash-frontend/src/components/block-loader/block-loader.tsx

```diff
@@ -166,11 +169,14 @@ export const BlockLoader: FunctionComponent<BlockLoaderProps> = ({
     }
 
     return {
-      ...(graphProperties as Required<BlockGraphProperties<true>["graph"]>),
+      ...graphProperties,
       blockEntity: {
         entityId: rootEntity.metadata.recordId.entityId,
         properties: rootEntity.properties,
       },
+      /* @todo - Why is this necessary? */
+      blockEntitySubgraph: graphProperties.blockEntitySubgraph,
+      readonly: graphProperties.readonly,
```

Comment:
Oh now that I see the diff I know why it's necessary 😅 . `eslint fix` on save must have removed it when the import was broken and it thought it was an `any` 🙃